app/controllers/comments_controller.rb

```diff
@@ -2,7 +2,7 @@ class CommentsController < ApplicationController
   def create
     @post = Post.find(params[:post_id])
     @comment = @post.comments.create(comment_parameters.merge(user_id: current_user.id))
-
+    flash[:notice] = 'Comment added successfully! '
     redirect_back(fallback_location: root_path)
   end
```

Comment:
- [ ] Adding a comment, I am getting this error, please fix it.

![setup-2](https://user-images.githubusercontent.com/55635977/145416152-c0cec300-8d44-4595-8cb9-9211bc74414a.png)